ui/src/controllers/renderDashboard.js

```diff
@@ -87,7 +87,7 @@ export default (req, res) => {
     })
   )
   .catch((err) => {
-    logger.error(err);
+    logger.error('Error in request', JSON.stringify(err));
     res.status(404).send(err.message);
```

Comment:
One of the issues was that the error message contained the text that you put in the URL which allowed html to be injected onto the page. Can you please change the response Chris to "Dashboard not found"?

Reply:
```suggestion
    res.status(404).send('Dashboard not found');
```